internal/components/components.go

```diff
@@ -230,6 +231,7 @@ func Components() (component.Factories, error) {
 		opencensusexporter.NewFactory(),
 		otlpexporter.NewFactory(),
 		otlphttpexporter.NewFactory(),
+		parquetexporter.NewFactory(),
```

Comment:
I am not sure if an exporter that has an incomplete implementation should be added to the components list?

I worry that someone may get excited about it existing then be sadden that it is currently a work in progress if this was to be merged.

Reply:
It's the way it is - first add the skeleton, then the impl. That's how it's been for other components. Maybe I missed something, feel free to correct me.